src/components/Table.re

```diff
@@ -0,0 +1,186 @@
+module Styles = {
+  open Css;
+
+  let wrapper =
+    style([
+      overflowX(auto),
+      width(`percent(100.0)),
+      boxSizing(borderBox),
+      padding(px(Theme.Spacing.base)),
+      backgroundColor(hex("fafafa")),
+    ]);
+
+  let header =
+    style([
+      marginBottom(px(Theme.Spacing.base)),
+      display(flexBox),
+      alignItems(center),
+    ]);
+  let title = style([flexGrow(1.0)]);
+  let search = style([display(flexBox), alignItems(center)]);
+  let searchInput =
+    style([
+      padding(px(5)),
+      marginLeft(px(10)),
```

Comment:
shouldn't these spacing be coming from AstroDesign to match the grid system there?

Reply:
alsdjfas